doc/specs/#117-Windows-Store-Support.md

```diff
@@ -0,0 +1,144 @@
+
+### YAML Spec updates
+<b>URL</b> the URL used for the store is not the same as the URL users and the developers expect.  Therefore this field is unnecessary.  
+<b>REQUIRED: NEW: StoreID</b> the Store ID will be used to identify the app. example: 9nblggh4nns1   
+<b>InstallerType</b> A new format, the WindowsStore type will be provided
+<b> InstallerType - Default </b> this needs to be defined, but essentially the author can specify if the user does not select anything... which URL to default to.
+<b>Homepage</b> The Homepage will be the URL with the bigid https://www.microsoft.com/en-us/p/app-installer/9nblggh4nns1  
+<B>NEW: TransactionType</b> TransactionType will be used to indicate what type of 
+<b>NEW: Privacy Policy</b>  
+<b>NEW: Terms of transaction</b>  
+<b>License terms</b>  
+<b>Architecture</b> The architecture will support the architectures supported by the store.  neutral will most likely be the default.
+<b>-prelease</b> In order to support flighing, the client will need to support only showing apps with out the prerelease flag AND enabling the user override on the command line.
+
+### Flighting
+The store provides the ability to flight apps to partners.  But you must be logged in with your MSA to get the update.  By default the Store App support will be for providing the user with the GA version.  
+
+#### MSA Credentials
+In order to support flighting, the Windows Package Manager would need to pass credentials to the Windows Store.  This is not in scope for this release, therefore only GA apps are enabled.  
+
+Developers wishing to flight store apps will need to create a different app and BigID in order to make it available for flighting.  Developers will use the -preview flag in the version to restrict access to those that want prerelease builds.
+
+#### GA Only
+We are generic and only pull down the latest GA version of the app.  
+
+## UI/UX Design
+Each Store app in the repo will be interacted with by the client like any other app by the Windows Package Manager.
+
+```
+PS C:\> winget search appinstaller
+Name                    Id                                Version
+----------------------------------------------------------------------
+AppInstallerFileBuilder Microsoft.AppInstallerFileBuilder 1.2020.211.0
+```
+And Install
+### Store App installation experience
+Installing the store app will provide the MSIX installation experience.  The Windows Store App will stream install.
+
+![](images\117-Download.png)
+
+
+## Manifests
+Developers will be able to submit their app by providing meta data that includes the store ID or URL to the store app.
+
+### Duplicate app entries
+With the addition of the store, we will have an increase in duplicate entries.  If we have 2 manifests with the same application, we should try to eliminate the duplicate entries by default.  Otherwise it will make it difficult to limit the selection to the one file.
```

Comment:
> we should try to eliminate the duplicate entries by default
Seems like we need to prioritize the support for multiple installer in one manifest. And enforce it in manual review/ reject the manifest in index creation?